servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Publisher.java

```diff
@@ -2664,6 +2664,36 @@ public final Publisher<T> skipWhile(Predicate<? super T> predicate) {
         return filter(FilterPublisher.skipWhileSupplier(predicate));
     }
 
+    /**
+     * After the first element of this {@link Publisher} is emitted pass it, and a {@link Publisher} representing the
```

Comment:
Consider starting with `Converts this {@link Publisher} to a {@link Single} that will contain the first element ...`, similar to other `first*` operators